### PR TITLE
Change etcd cert inside flanneld job

### PR DESCRIPTION
**Pull request by lubronzhan**
**What this PR does / why we need it**:

**How can this PR be verified?**
Pipeline
**Is there any change in kubo-deployment?**
yes
**Is there any change in kubo-ci?**
yes
**Does this affect upgrade, or is there any migration required?**

**Which issue(s) this PR fixes:**
https://www.pivotaltracker.com/story/show/163786805
**Release note**:

```release-note

```

### CODE REVIEW

File: jobs/flanneld/templates/config/etcd-client.crt.erb [hunk 1]
```diff
@@ -1,2 +1,2 @@
-<%= link('etcd').p('tls.etcdctl.certificate') %>
+<%= link('etcd').p('tls.etcdctl-flanneld.certificate') %>
```
Comment (tvs) [inline jobs/flanneld/templates/config/etcd-client.crt.erb]: Let's kill the link here and just let it be consumed directly. The link doesn't really give us anything here - to be entirely honest, its just led us to re-use certs in a way we shouldn't be in the first place.

Comment (pydctw) [reply jobs/flanneld/templates/config/etcd-client.crt.erb]: cool

File: jobs/flanneld/spec [hunk 2]
```diff
@@ -20,6 +20,12 @@ properties:
...
     default: "vxlan"
+  tls.etcdctl.ca:
+    description: CA for etcdctl client authentication
+  tls.etcdctl.certificate:
+    description: Certificate for etcdctl client authentication
+  tls.etcdctl.private_key:
+    description: Private key for etcdctl client authentication
 
 consumes:
 - name: etcd
```
Comment (tvs) [inline jobs/flanneld/spec]: If I'm not mistaken, we can now remove the `consumes` link for `etcd`

Comment (lubronzhan) [reply jobs/flanneld/spec]: It still use etcd link for dns suffix
https://github.com/cloudfoundry-incubator/kubo-release/blob/4906d3a192584924a0ab85a25dafcc047af36a28/jobs/flanneld/templates/bin/flanneld_ctl.erb#L9

Comment (lubronzhan) [reply jobs/flanneld/spec]: And here https://github.com/cloudfoundry-incubator/kubo-release/blob/4906d3a192584924a0ab85a25dafcc047af36a28/jobs/flanneld/templates/bin/flanneld_ctl.erb#L39

Should we input them all as variable?

**Pull request merged**